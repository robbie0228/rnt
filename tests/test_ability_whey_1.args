-ability1 WWALF
-ability2 WWALF
