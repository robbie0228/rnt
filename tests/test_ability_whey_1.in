board
abilities
ability 1 A
ability 1 a
board
move a down
abilities
ability 1 a
ability 1 A
board
move A up
board
ability 2 a
move a down
board
ability 2 A
move A up
board
ability 4 a
move a down
board
move A up
board
quit
