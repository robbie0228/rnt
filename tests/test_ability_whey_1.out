Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1's Abilities:
1: Whey, Unused
2: Whey, Unused
3: Ambush, Unused
4: Link Boost, Unused
5: Firewall, Unused

Cannot use Whey on a link that is not yours!

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: V2   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 2's Abilities:
1: Whey, Unused
2: Whey, Unused
3: Ambush, Unused
4: Link Boost, Unused
5: Firewall, Unused

Cannot use Whey on a link that is not yours!

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 4
A: V2   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: V2   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
a..de...
........
........
........
........
A..DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 4
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 3
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
a.......
........
........
........
A..DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 4
A: V2   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 3
a: V3   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
a.......
........
........
A.......
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 3
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 2
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
........
........
a.......
A.......
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 3
A: V3   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 2
a: V3   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
........
........
A.......
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 1V
Abilities: 3
A: V3   B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

